README.md

```diff
@@ -0,0 +1,83 @@
+services automatically. It’s intended to be used as a part of your CI to simplify the development process, improve your 
+security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
+
+## Supported CI Services
+### GithubActions
+Example using the NightfallDLP Github Action inside a Github Workflow  
+_(**N.B.** you must use the actions/checkout step before the running the nightfalldlp action in order for it to function properly)_
+```yaml
+name: nightfalldlp
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  run-nightfalldlp:
+    name: nightfalldlp
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo Action
+        uses: actions/checkout@v2
+
+      - name: nightfallDLP action step
+        uses: nightfallai/nightfall_dlp_action@CreateAction
+        env:
+          NIGHTFALL_API_KEY: ${{ secrets.NIGHTFALL_API_KEY }}
+          EVENT_BEFORE: ${{ github.event.before }}
+```
+
+### Action Spec  
+NightfallDLP requires a config file and a few Environment variables in order to run  
+
+**Config File (detectors)**  
+ - place a `.nightfalldlp/` directory within the root of your target repository, and inside it a `config.json` file
+ in which you can configure your detectors (see `Detectors` section below for more information on Detectors)  
+ - inside the `detectors` map the keys are Detector Names and the values are the minimum likelihood for a detector to be triggered 
+
+ - sample `.nightfalldlp/config.json` file
+```json
+{
+    "detectors": {
+        "CREDIT_CARD_NUMBER": "POSSIBLE",
+        "PHONE_NUMBER": "LIKELY"
+    }
+}
+```
+**Env Variables**      
+These variables should be made available to the nightfall_dlp_action by adding them to the `env:` key in your workflow  
+1) `NIGHTFALL_API_KEY`
+    - get a (FREE) Nightfall AI DLP Scan API Key by registering an account with the Nightfall API HERE
+    - add this variable to your target repository's "Github Secrets" and passed in to your Github Workflow's `env`.
+2) `GITHUB_TOKEN`
+    - this is automatically injected by Github inside each Workflow (via the `secrets` context), you just need to set it 
+    to the env key
+
+3) `EVENT_BEFORE` (*only required for Github Workflows running on a `push` event)
+    - the value for this var lives on the `github` context object in a Workflow - EVENT_BEFORE should always point to
+    `${{ github.event.before }}` (as seen in the example above)
+    
+**Supported Github Events**  
+NightfallDLP can run in a Github Workflow triggered by the following events:
+1) PULL_REQUEST
+2) PUSH
+
+NightfallDLP is currently unable to be used in forked Github repositories due to Github's disabling of secrets sharing when Workflows originate from forks.
+
+## Detectors
+Each detector represents a type of information you want to search for in your code scans (e.g. CRYPTOGRAPHIC_KEY). The 
```

Comment:
We have a glossary of these detectors published in https://developer.nightfall.ai/reference#glossary ... (not sure how to tag Isaac, his name isn't auto-completing), but @Isaac, should we consider disabling the password on the developer.nightfall.ai docs and linking it to here in these public docs?

Either that, or we should have a duplicate resource that enumerates the detectors we scan with.

Reply:
yeah great question. I think it's best to keep that detectors list inside the API docs and link to them from here (so we don't have to maintain this list in separate places, to keep this Readme as Github Actions focused as possible, etc.), so would be down to lift the password requirements for those.

Reply:
I think you added the wrong isaac @evanfuller

Reply:
@isaac243